README.md

```diff
@@ -27,6 +27,11 @@ Examples:
 
 `url` is required and the file must exist or an error is given.  SublimeUrl expects to find Sublime Text in `/Applications/Sublime Text.app`.
 
+## Homestead mapping to MacOS
```

Comment:
I'm accepting this feature, but not specifically for Homestead - in fact, I don't know what that it is; although I understand it's your own use case, I think this has broader utility. The config structure will change a lot before next release so I hope you won't be offended if I rewrite this section completely later.

Reply:
@inopinatus of course not!

Actually that was a quick and dirty approach, that does not handle multiple mappings.

Homestead is a virtual environment that allows you to develop laravel projects on your laptop without loading it with a bunch of software. You then can get rid of the VM and the laptop is "clean".